eno-ws/src/main/java/fr/insee/eno/ws/controller/exception/EnoExceptionController.java

```diff
@@ -0,0 +1,29 @@
+package fr.insee.eno.ws.controller.exception;
+
+import fr.insee.eno.legacy.exception.EnoGenerationException;
+import fr.insee.eno.legacy.exception.EnoParametersException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+
+@ControllerAdvice
```

Comment:
could be great to have a formatted error response in json

Reply:
Noted, there's work on exception handling to do, I'll do it at that moment